### DISCUSSION TITLE
Trying to start storybook from node

### DISCUSSION THREAD

**Discussion by isuro** [storybookjs/storybook#27892] [Documentation] [answered]
Hey all! I'm working on migrating a large monorepo from 6.x to latest. One functionality that was being used before was a wrapper around Storybook's cli commands that use the node api to serve or build. Is there any equivalent to this?

**Comment by Marklb** [comment, marked as answer]
If I understand what you are asking, you aren't wanting to wrap the cli command, but are wanting to bypass them and call the same function that those parsed arguments would be passed to, right?

The build function that is exported from `@storybook/core-server` is what I think you want. https://github.com/storybookjs/storybook/blob/17b7512c60256c739b890b3d85aaac992806dee6/code/lib/core-server/src/standalone.ts#L5

I don't know if there is any documentation for doing that, but the Angular builders bypass the cli and may be useful as an example. https://github.com/storybookjs/storybook/blob/17b7512c60256c739b890b3d85aaac992806dee6/code/frameworks/angular/src/builders/start-storybook/index.ts#L198
